packages/testnet-delegation/contracts/mocks/MockIDPOS.sol

```diff
@@ -0,0 +1,88 @@
+// (c) 2023-2024, Taraxa, Inc. All rights reserved.
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+interface MockIDPOS {
```

Comment:
Why do we have this file if we already have the `[IDPOS.sol](https://github.com/Taraxa-project/taraxa-platform/pull/427/files#diff-9e867ada1ee36ff0543f30e387f23be30403b1a164021467de2454a7d2104d5c)` one?

Reply:
Ah, I see now, this is only a part of the main interface with the necessary business logic for tests, that's why it was duplicated.